### ISSUE TITLE
Incorrect root page link for 'Modelica Standard Library'

### ISSUE THREAD

**Issue by henrikt-ma** [modelica/www.modelica.org#49]
The link here is the same as the link for _Modelica Libraries_, and needs to be fixed:
https://github.com/modelica/www.modelica.org/blob/3f2aa3a2a40fc7d421d296c501a9a61ee97f285d/content/_index.html#L30

**Comment by HansOlsson** [comment]
To me the problem seems to be more fundamental:
We don't have a web-page for the Modelica Standard Library.

One possibility would be linking directly to doc.modelica.org or (better) https://doc.modelica.org/Modelica%204.0.0/Resources/helpDymola/Modelica_UsersGuide.html#Modelica.UsersGuide.Overview - but I don't think that overview is an ideal overview for newcomers.

**Comment by dietmarw** [comment]
Why not simply linking to the main page https://doc.modelica.org/Modelica%204.0.0/Resources/helpDymola/Modelica.html ?

**Comment by henrikt-ma** [comment]
It is also not ideal to link to a tool-specific page.

**Comment by beutlich** [comment]
> One possibility would be linking directly to doc.modelica.org

Yes, that is the landing page for MSL, see https://github.com/modelica/ModelicaStandardLibrary/issues/3287.

If you prefer, it can be renamed to library.modelica.org to be consistent with specification.modelica.org.

**Comment by henrikt-ma** [comment]
It doesn't make sense to rename it _library.modelica.org_ as long as the first part of the page is _Modelica Specification_.

It would be nice if https://doc.modelica.org/Modelica%204.0.0/ was a tool-independent landing page for the library.  Then this would also be what the _Modelica Standard Library Documentation_ section of https://doc.modelica.org should point to (independently of tool selected in the menu).  The menus for selecting library version and tool would then make more sense at the library landing page, and at _library.modelica.org_ I would suggest that we omit the library selection menu, and instead have a section called _Library Documentation_ where there is a table with links to the landing page for each library.

**Comment by beutlich** [comment]
> It doesn't make sense to rename it _library.modelica.org_ as long as the first part of the page is _Modelica Specification_.

Guess what, this shall be removed then. doc.modelica.org is legacy in that sense that it both refers to library doc and specification.modelica.org.

**Comment by henrikt-ma** [comment]
Remove the section or the entire page?  What would be the problem with a common landing page with sections both for specification and each of the libraries?

**Comment by beutlich** [comment]
Hm, I thought this ticket is about a landing page for MSL alone.

I am in favour of having one (and tool-independent) MSL documentation page.

**Comment by dzimmer** [comment]
This ticket was originally about fixing a link. 

I agree with @dietmarw. We shall simply link to the best page that we currently have and that seems:
https://doc.modelica.org/Modelica%204.0.0/Resources/helpDymola/Modelica.html

That it was generated by a specific tool, is not so relevant here. The content is tool-independent.
We can still change the link, once we have a better page to link to.

**Comment by henrikt-ma** [comment]
Please make sure there is an issue about creating a better landing page before this is closed.

**Comment by casella** [comment]
> I am in favour of having one (and tool-independent) MSL documentation page.

I agree. Of course we will have to use one specific tool to actually generate the content, but that should be inessential in how we present it..

**Comment by henrikt-ma** [comment]
The landing page should not be generated by any Modelica tool.  The page have an official MA look, and provide links to generated MSL documentation for any tool that wants presence on this page.

**Comment by beutlich** [comment]
Meanwhile, the reference to 'Modelica Standard Library' got removed entirely by 279e67330da5318aa899a0f1c816150a662ab86b.
